manifest.json

```diff
@@ -419,11 +419,11 @@
         }
       },
       "midiJs": {
-        "version": "2ef687b47e5f478f1506b47238f3785d9ea8bd25",
+        "version": "a8a84257afa70721ae462448048a87301fc1554a",
         "downloadFormat": "zip",
-        "url": "https://github.com/mudcube/MIDI.js/archive/2ef687b47e5f478f1506b47238f3785d9ea8bd25.zip",
+        "url": "https://github.com/mudcube/MIDI.js/archive/a8a84257afa70721ae462448048a87301fc1554a.zip",
         "rootDirPrefix": "MIDI.js-",
-        "targetDir": "midi-js-2ef687"
+        "targetDirPrefix": "MIDI.js-"
```

Comment:
Please use `targetDir` field specifying the commit, as in the previous pattern.

Reply:
I don't think this has been addressed yet?

Reply:
Hi @apb7, sorry missed has been addressed now. Thanks!